scripts/local_five_node_staking_leveldb.lua

```diff
@@ -0,0 +1,11 @@
+cmds = {
+    "startnode node1 --client-location /Users/manthandave/development/goprojects/src/github.com/ava-labs/avalanchego-internal/build/avalanchego --db-type=leveldb --db-dir=./db --keystore-db-type=leveldb --keystore-db-dir=./db/ksdb --staking-enabled=true --http-port=9650 --staking-port=9651 --log-level=debug --bootstrap-ips= --staking-tls-cert-file=certs/keys1/staker.crt --staking-tls-key-file=certs/keys1/staker.key",
```

Comment:
Can you remove the client location flag here or switch from using the current path to something that uses `$GOPATH/...` ?

Reply:
removed client location